Python/compile.c

```diff
@@ -9230,50 +9230,36 @@ optimize_basic_block(PyObject *const_cache, basicblock *bb, PyObject *consts)
     return -1;
 }
 
-static bool
-basicblock_has_lineno(const basicblock *bb) {
```

Comment:
We still need to propagate line numbers to returns and raises, so where did this functionality go?

Reply:
This function was never used for that. It was added in bde06e1b8381f140b296a397ddd1deb1c784ff8e.

Reply:
:+1: